src/latex.jl

```diff
@@ -20,6 +23,7 @@ Can be added to `Luxor.jl` graphics via [`Video`](@ref).
 - **The `latex` method must be called from within an [`Object`](@ref).**
 
 # Arguments
+Positional:
```

Comment:
please follow the style guide for these docstrings as listed here:
https://github.com/invenia/BlueStyle#keyword-arguments

They can also be seen in some other methods like our `render` function